packages/synthetic-chain/src/cli/cli.ts

```diff
@@ -117,7 +120,21 @@ switch (cmd) {
         console.log(chalk.cyan.bold(`Testing ${proposal.proposalName}`));
         const image = imageNameForProposal(proposal, 'test');
         bakeTarget(image.target, values.dry);
+
+        const proposalPath = `${root}/proposals/${proposal.path}`;
+
+        if (fileExists(`${proposalPath}/pre_test.sh`))
+          execSync(`/bin/bash ${proposalPath}/pre_test.sh`, {
```

Comment:
the scripts have shebang. the caller should not be concerned with how they're interpreted.
```suggestion
          execSync(`${proposalPath}/pre_test.sh`, {
```